### DISCUSSION TITLE
Habit plane determination

### DISCUSSION THREAD

**Discussion by wanghaijian217** [ORTools4MTEX/ORTools#60] [Q&A] [answered]
Hi Frank and Azdiar !
Thank you for your contribution in the process of determining the Habit plane
However, I need to determine the Habit plane of medium carbon steel at present, and my own data is EBSD data of quenched samples at present. My purpose was to determine which Habit plane my own medium carbon steel was inclined to, and what was the error, but the actual calculation was not the same as that in your paper: Why?
Isn't the following picture an oriented triangle, which doesn't mean that it is biased towards the (111) Habit plane?
How do I determine the habitual surface of steel through your code, compare (111), (252), (575) and so on？
![image](https://github.com/ORTools4MTEX/ORTools/assets/131143952/84845832-5f19-4a0a-9776-d06514050b60)
![image](https://github.com/ORTools4MTEX/ORTools/assets/131143952/22b683bb-5b75-4bd4-9470-71be468e7b79)
![image](https://github.com/ORTools4MTEX/ORTools/assets/131143952/0efc8fd0-9e58-4b35-8c5e-88db6f65e1c6)
![image](https://github.com/ORTools4MTEX/ORTools/assets/131143952/d36805b1-b602-40ba-ad55-ed6b010e7282)

**Comment by AzdiarGazder** [comment, marked as answer]
Hi @wanghaijian217,

As always, you are welcome to the functions created in ORTools.

Here are some answers and pointers based on the information you have provided:

- Please note that you have not provided the script or map data. Consequently, I do not know exactly how the habit plane was  calculated. While the IPF shows that the habit plane is close to ~{111}, of greater concern is the rather poor statistics seen in the spherical projection of contours of the fitted traces and the habit plane. It is not a good fit - and the cause could be any number of reasons as stated in the paper.
The statistical data outputted from the [computeHabitPlane](https://github.com/ORTools4MTEX/ORTools/blob/develop/src/computeHabitPlane.m) function shows that you may also have relaxed a number of the constraints that are imposed by default by the function. This may be erroneous. Please be careful with this function as the calculation requires a good amount of statistics and a strict set of function constraints to return a robust answer. EBSD maps with only a few parent austenite grains, or with poor spatial resolution, or with relaxed function constraints may return an incorrect habit plane.

- Our paper specifically dealt with martensite. We increased the carbon content of the base alloy in order to change the morphology of the martensite in steels from laths to plates, which as per theory is also supposed to change the habit plane. That is why we discussed (111), (252), and (575) in Fig. 14 of the paper. If that is your intention, you can simply add those points to your IPF as we did.

Hope this helps.

<ins>**Lastly, and importantly, if an explanation from the maintainers resolves your issue, please press the "Mark as answer" button on the appropriate response. Your help in this regard enables us to close discussions as complete. **</ins>

Warm regards,
Azdi
